bigtable/google/cloud/bigtable/instance.py

```diff
@@ -481,3 +483,99 @@ def list_app_profiles(self):
         """
         resp = self._client._instance_admin_client.list_app_profiles(self.name)
         return [AppProfile.from_pb(app_profile, self) for app_profile in resp]
+
+    def get_iam_policy(self):
+        """Gets the access control policy for an instance resource.
+
+        .. code-block:: python
+
+            from google.cloud.bigtable.client import Client
+            from google.cloud.iam import Policy
+
+            client = Client(admin=True)
+            instance = client.instance('[INSTANCE_ID]')
+            policy_latest = instance.get_iam_policy()
+            print (policy_latest.owners)
+
+        :rtype: :class:`~google.cloud.iam.Policy`
+        :returns: A :class:`~google.cloud.iam.Policy` instance.
+        """
+        instance_admin_client = self._client._instance_admin_client
+        resp = instance_admin_client.get_iam_policy(resource=self.name)
+        return Policy.from_api_repr(self._to_dict_from_policy_pb(resp))
+
+    def set_iam_policy(self, policy):
+        """Sets the access control policy on an instance resource. Replaces any
+        existing policy.
+
+        .. code-block:: python
+
+            from google.cloud.bigtable.client import Client
+            from google.cloud.iam import Policy
+
+            client = Client(admin=True)
+            instance = client.instance('[INSTANCE_ID]')
+            policy = instance.get_iam_policy()
+            policy.owners = [
+                Policy.user("test_iam@test.com"),
+                Policy.service_account("sv_account@gmail.com")]
```

Comment:
An observation, the code example above will not work since Bigtable `Instance` resource only supports predefined roles: `'roles/bigtable.admin'`, `'roles/bigtable.reader'`, `'roles/bigtable.user'` and `'roles/bigtable.viewer'`.
The correct way would be 
```python
policy['roles/bigtable.admin'] = [
    policy.user("test_iam@test.com"),
    policy.service_account("sv_account@gmail.com")]
result = instance.set_iam_policy(policy)
print (result['roles/bigtable.admin'])
```
To enhance user's experience I am suggesting to add `policy.py` (gist [link](https://gist.github.com/AVaksman/66d1219a338f73dcf02cfb2b6bb2effa)) that extends `iam.Policy` to this library and reroute `set_iam_policy` method through it.
Then the workflow will change to 
```python
from google.cloud.bigtable.client import Client
from google.cloud.bigtable.policy import BIGTABLE_ADMIN_ROLE
from google.cloud.bigtable.policy import Policy

    client = Client(admin=True)
    instance = client.instance('[INSTANCE_ID]')
    policy = Policy()
    policy[BIGTABLE_ADMIN_ROLE] = [
        policy.user("test_iam@test.com"),
        policy.service_account("sv_account@gmail.com"),
    ]
    result = instance.set_iam_policy(policy)
    print (result.bigtable_admins)
```

Reply:
Done, thanks.